sdk/txbuild/create_escrow.go

```diff
@@ -35,8 +36,12 @@ func CreateEscrow(p CreateEscrowParams) (*txnbuild.Transaction, error) {
 		},
 	}
 	if !p.Asset.IsNative() {
+		cta, err := p.Asset.ToChangeTrustAsset()
```

Comment:
💡 I'd call `MustToChangeTrustAsset` instead. We only support native assets and credit assets, and both of these assets will never return an error from that other function.

```suggestion
		cta, err := p.Asset.MustChangeTrustAsset()
```